docker-compose.manager.yml

```diff
@@ -0,0 +1,46 @@
+services:
+  manager:
+    image: ragapp/manager
+    build:
+      context: src/manager
+      args:
+        # Build time variables to set the basePath for static files
+        - APP_PATH=/manager
```

Comment:
needed? don't we rewrite them on startup?

Reply:
Ah, this one is now redundant. Sorry for the confusion